erts/emulator/beam/jit/x86/instr_bs.cpp

```diff
@@ -1596,8 +1638,200 @@ struct BscSegment {
 
     Uint error_info;
     Sint effectiveSize;
+    enum class action { NONE, ACCUMULATE_FIRST, ACCUMULATE, STORE } action;
```

Comment:
Would you mind adding a small description what the different actions do and when they trigger?

Reply:
I have renamed `NONE` to `DIRECT` and added more comments.